sample_apps/run_example.sh

```diff
@@ -346,6 +365,15 @@ function rm_non_git_files() {
     run_cmd
     run_pushd "${CERT_PROTO}"
 
+    local MbedTLS_dir_exists=0
+    # Use full dir-path so this works cleanly under --dry-run mode, too.
+    if [ -d "${CERT_PROTO}/${Simple_App_under_gramine_MbedTLS_dir}" ]; then
+        set -x
+        mv "${CERT_PROTO}/${Simple_App_under_gramine_MbedTLS_dir}" /tmp
```

Comment:
This fix is to handle the case where configure-MbedTLS was done earlier (either by this script or the user did it manually).

Then, we don't want this `rm_non_git_files` to `rm` that mbedtls/ dir and its contents. Otherwise, we will end up re-downloading stuff from the net on next execution ... which is sub-optimal.